Backend-tests.md

```diff
@@ -151,33 +151,57 @@ In fact, too many mocks can be a problem because when someone changes the code y
 
 You can run backend tests like this:
 
+Python:
 ```console
 python -m scripts.run_backend_tests
 ```
 
+Docker:
+```console
+make run_tests.backend
```

Comment:
this command is throwing an error in as speicified in below image but that will come as docker setup is in building progress as said by praneeth 
![Screenshot from 2024-02-08 18-11-21](https://github.com/oppia/oppia-web-developer-docs/assets/127002519/861f9514-8b6b-4de4-9e41-29fdfa636038)

Reply:
This error from backend tests can be ignored for now.